recipes/default.rb

```diff
@@ -85,19 +86,19 @@
   end
 
   yum_repository 'webtatic' do
-    description (node['librenms']['repo_webtatic']['desc']).to_s
-    baseurl (node['librenms']['repo_webtatic']['url']).to_s
+    description node['librenms']['repo_webtatic']['desc']
+    baseurl node['librenms']['repo_webtatic']['url']
     gpgcheck false
     enabled true
-    only_if { node.normal['librenms']['repo_webtatic']['enabled'] = 'true' }
+    only_if { node['librenms']['repo_webtatic']['enabled'] }
```

Comment:
You should only use this attribute for the enabled property:
```
enabled node['librenms']['repo_webtatic']['enabled']
```

Reply:
done.